<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gestion_utilisateurs</class>
 <widget class="QMainWindow" name="gestion_utilisateurs">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>90</y>
      <width>601</width>
      <height>371</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">


*{
font-family:century gothic;
background:#333;
font-size:14px;

}
QGroupBox
{
background:#333;
background-image:url(:/selem2.jpg);
}
#pushButton_ajoute
{
background:#3366cc;
border-radius:15px;
}
#pushButton_ajoute:hover
{
color:#333;
border-radius:15px;
background:white;
}
#pushButton_rechercher_2
{
background:#3366cc;
border-radius:15px;
}
#pushButton_rechercher_2:hover
{
color:#333;
border-radius:15px;
background:white;
}
#pushButton_rechercher_3
{
background:#3366cc;
border-radius:15px;
}
#pushButton_rechercher_3:hover
{
color:#333;
border-radius:15px;
background:white;
}
#pushButton_2
{
background:#3366cc;
border-radius:15px;
}
#pushButton_2:hover
{
color:#333;
border-radius:15px;
background:white;
}
#pushButton_rechercher_4
{
background:#3366cc;
border-radius:15px;
}
#pushButton_rechercher_4:hover
{
color:#333;
border-radius:15px;
background:white;
}
#pushButton_modifer
{
background:#3366cc;
border-radius:15px;
}
#pushButton_modifer:hover
{
color:#333;
border-radius:15px;
background:white;
}
#pushButton_impression_client
{
background:#3366cc;
border-radius:15px;
}
#pushButton_impression_client:hover
{
color:#333;
border-radius:15px;
background:white;
}
#pushButton_rechercher
{
background:#3366cc;
border-radius:15px;
}
#pushButton_rechercher:hover
{
color:#333;
border-radius:15px;
background:white;
}

</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Ajouter</string>
     </attribute>
     <widget class="QLineEdit" name="lineEdit_id">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>70</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:white;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_prenom">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>110</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:white;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_nom">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>150</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:white;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_age">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>190</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:white;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_id">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>70</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>ID</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_prenom">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>100</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Prenom</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_nom">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>150</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Nom</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_age">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>180</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Age</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_ajoute">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>260</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_pixmap">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>30</y>
        <width>141</width>
        <height>111</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QTableView" name="tableView_recherche_2">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>220</y>
        <width>361</width>
        <height>101</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:white;
</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_rechercher_2">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>170</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>rechercher</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Supprimer</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>84</x>
        <y>250</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>supprime</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>90</y>
        <width>47</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>ID</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_id_supprimer">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>90</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:white;</string>
      </property>
     </widget>
     <widget class="QTableView" name="tableView_recherche_3">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>160</y>
        <width>331</width>
        <height>101</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:white;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_rechercher_3">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>112</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>rechercher</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>modifer</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_modifer">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>240</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>modifer</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_id_modifer">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>40</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:white;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_prenom_modifer">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>80</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:white;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_nom_modifer">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>130</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:white;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_age_modifer">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>180</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:white;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>40</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>ID</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>80</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>prenom</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>36</x>
        <y>130</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>nom</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>170</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>age</string>
      </property>
     </widget>
     <widget class="QTableView" name="tableView_recherche_4">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>210</y>
        <width>331</width>
        <height>91</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:white;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_rechercher_4">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>110</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>rechercher</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>afficher</string>
     </attribute>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>451</width>
        <height>271</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:white;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_impression_client">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>190</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>impression</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>80</y>
        <width>70</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>id</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_2">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>100</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>prenom</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_3">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>130</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>nom</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_4">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>150</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>age</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>50</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>trier par</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>rechercher</string>
     </attribute>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>19</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>rechercher par id:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_recherche">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>30</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:white;</string>
      </property>
     </widget>
     <widget class="QTableView" name="tableView_recherche">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>451</width>
        <height>192</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:white;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>70</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>rechercher:</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>120</y>
        <width>82</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>ID</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_2">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>160</y>
        <width>131</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>PRENOM</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_3">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>200</y>
        <width>82</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>NOM</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_4">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>240</y>
        <width>82</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>AGE</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>470</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:white;</string>
    </property>
    <property name="text">
     <string>Quitter</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>0</y>
      <width>811</width>
      <height>581</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../Desktop/5.jpg</pixmap>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>711</width>
      <height>31</height>
     </rect>
    </property>
    <property name="title">
     <string>Gestion des utilisateurs</string>
    </property>
   </widget>
   <zorder>label_9</zorder>
   <zorder>tabWidget</zorder>
   <zorder>pushButton</zorder>
   <zorder>groupBox</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
